core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/mariadb/MariadbDatabaseConnectionService.java

```diff
@@ -28,4 +30,8 @@ public String getDriver(MariadbDatabaseConnection databaseConnection) {
         return MariadbDatabaseConnection.class;
     }
 
+    @Override
+    public String refactorLimitAndOffset(MariadbDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class